pkg/pillar/cas/cas.go

```diff
@@ -108,7 +108,7 @@ type CAS interface {
 	// if there is an exception while reading the blob data.
 	//NOTE: This either loads all the blobs or loads nothing. In other words, in case of error,
 	// this API will GC all blobs that were loaded until that point.
-	IngestBlobsAnsCreateImage(reference string, blobs ...*types.BlobStatus) error
+	IngestBlobsAnsCreateImage(reference string, blobs ...*types.BlobStatus) ([]*types.BlobStatus, error)
```

Comment:
`Ans`? Is that supposed to be `And`? Would need to fix in the comment as well.

Reply:
And anywhere that consumes it, which I think is just in one place in `volumemgr/updatestatus.go`, but also the `log.Infof()` lines.

Reply:
BTW, this totally can go in a separate PR, but if you already are touching it, might as well fix it.

Reply:
Corrected.